docs/experimental-features.md: typo (#7062)
ipfs/go-ipfs: docs/experimental-features.md

Gossipsub is a new, experimental routing protocol for pubsub that

> should waste less bandwidth than floodsub, the current pubsub

Reviewer: note: british english (but normalizing is fine)

Author: yeah I know, should have added this to the changelog

### State
Experimental

> Allows to create directories with an unlimited number of entries - currently

Reviewer 2: ```suggestion
Allows creating directories with an unlimited number of entries - currently
```

Experimental, default-disabled.

> Utilizes pubsub for publishing ipns records in real-time.

Reviewer 2: ```suggestion
Utilizes pubsub for publishing ipns records in real time.
```
in the noun form, two words is preferred: https://www.merriam-webster.com/dictionary/real%20time

When it is enabled:
- IPNS publishers push records to a name-specific pubsub topic,
  in addition to publishing to the DHT.
- IPNS resolvers subscribe to the name-specific topic on first

>   resolution and receive subsequently published records through pubsub in real-time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.

Reviewer 2: ```suggestion
  resolution and receive subsequently published records through pubsub in real time. This makes subsequent resolutions instant, as they are resolved through the local cache. Note that the initial resolution still goes through the DHT, as there is no message history in pubsub.
```
https://www.merriam-webster.com/dictionary/real%20time

ipfs config --json Experimental.QUIC true
```

For listening on a QUIC address, add it to the swarm addresses, e.g. `/ip4/0.0.0.0/udp/4001/quic`.

### Road to being a real feature

> - [ ] The IETF QUIC specification needs to be finalized.

Reviewer: (also british english)

### Road to being a real feature

- [ ] needs real-world testing
- [ ] needs adoption

> - [ ] needs to support all provided features

Reviewer: ```suggestion
- [ ] needs to support all provider subsystem features
```
